README.md

```diff
@@ -27,8 +27,7 @@ For deployment of NMO using these images you need
 Then run `operator-sdk run bundle quay.io/kubevirt/node-maintenance-operator-bundle:latest`
 
 ### Build and deploy from sources
-
-Follow the instructions on https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/#run-the-operator
+Run `make bundle bundle-build bundle-push` to build and push the bundle, and then `operator-sdk run bundle BUNDLE_IMG` where BUNDLE_IMG is the newly create bundle image (based on the instructions [here](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/#run-the-operator)).
```

Comment:
>Isn't your added text the same what you find on the linked website? 
 
Yes, I was trying to be more specific because in https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/#3-deploy-your-operator-with-olm it suggests to install OLM as well, which is not needed once you have Openshift.
Maybe it is better to unchnage it, because one could use in Kubernetes cluster and not an Openshift.